framework/auth/campaigns.py

```diff
@@ -9,7 +9,7 @@
 CAMPAIGNS = ImmutableDict({
     'prereg': {
         'system_tag': 'prereg_challenge_campaign',
-        'redirect_url': lambda: web_url_for('prereg_landing_page'),
+        'redirect_url': lambda: web_url_for('prereg_landing_page', _absolute=True),
```

Comment:
@mfraezz Thanks for bringing this up and it is an issue.  Several login logic especially for campaigns has been changed in this PR. I can pull and update ERPC as well. Amanda showed me an issue with ERPC. Looks like that ERPC follows Prereg, which makes it easy for me apply this new logic to it as well.